docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
 
-**read_only**==*true*|*false*
-  Run every container in read-only mode. Automatically mount tmpfs on `/run`, `/tmp` and `/var/tmp`.
-  Setup images to run in read-only. (default: false)
 
 ## CRIO.IMAGE TABLE
+The `crio.image` table contains settings pertaining to the management of OCI images.
 
-**default_transport**
-  A prefix to prepend to image names that can't be pulled as-is (default: "docker://")
+**default_transport**="docker://"
+  A prefix we for images that cannot be parsed as a valid source reference.
 
-**image_volumes**=""
-  Image volume handling ('mkdir', 'bind' or 'ignore') (default: "mkdir")
-  mkdir: A directory is created inside the container root filesystem for the volumes.
-  bind: A directory is created inside container state directory and bind mounted into
-  the container for the volumes.
-  ignore: All volumes are just ignored and no action is taken.
+**pause_image**="k8s.gcr.io/pause:3.1"
+  The image used to instantiate infra containers.
 
-**insecure_registries**=""
-  Enable insecure registry  communication,  i.e.,  enable  un-encrypted
-  and/or untrusted communication.
+**pause_command**="/pause"
+  The command to run in a `pause_image` to have a container just sit there. If the image contains the necessary information, this value does not need to be specified.
 
-  List  of  insecure registries can contain an element with CIDR notation
-  to specify a whole  subnet.  Insecure  registries  accept  HTTP  and/or
-  accept HTTPS with certificates from unknown CAs.
+**signature_policy**="/etc/containers/policy.json"
+  Path the file which decides what sort of policy we use when deciding whether or not to trust an image that we've pulled. Please refer to containers-policy.json(5) for more details.
 
-  Enabling  --insecure-registry  is useful when running a local registry.
-  However, because its use creates  security  vulnerabilities  it  should
-  ONLY  be  enabled  for testing purposes.  For increased security, users
-  should add their CA to their system's list of trusted  CAs  instead  of
-  using --insecure-registry.
+**image_volumes**="mkdir"
+  Controls how image volumes are handled. The valid values are mkdir, bind and ignore; the latter will ignore volumes entirely.
 
-**pause_command**=""
-  Path to the pause executable in the pause image (default: "/pause")
+**insecure_registries**=[]
+  List of registries to skip TLS verification when pulling images.
 
-**pause_image**=""
-  Image which contains the pause executable (default: "kubernetes/pause")
+**registries**=["docker.io"]
+  List of registries to be used when pulling an unqualified image (e.g., "alpine:latest").
+
+CRI-O reads its configured registries defaults from the system wide containers-registries.conf(5) located in /etc/containers/registries.conf. If you want to modify just CRI-O, you can change the registies configuration in this file. Otherwise, leave `insecure_registries` and `registries` comment out to use system's defaults.
```

Comment:
suggest: "comment out to use system's defaults." to "commented out to use the system's defaults from /etc/containers/registries.conf."